Dockerfiles/Dockerfile.smoke_test

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.10.3
+
+RUN openssl genrsa -out /var/local/private.pem 2048
```

Comment:
I don't think the client needs this key pair. Can be removed.

Reply:
You are right.  Removed.